src/index.ts

```diff
@@ -80,8 +83,6 @@ export const DEFAULT_GLOBAL_SHIMS_PRESET: GlobalShim[] = [
   },
 ]
 
-export const MTS_SUPPORTED_NODE_VERSION = 16
```

Comment:
Why are these constants removed? The name here is important to understand what are they and why are magic numbers for.